kernel/packages/shared/comms/browser.ts

```diff
@@ -0,0 +1,6 @@
+export function isWebGLCompatible() {
+  // Create canvas element. The canvas is not added to the document itself, so it is never displayed in the browser window.
+  var canvas = <HTMLCanvasElement>document.createElement("canvas");
+  var gl = canvas.getContext("webgl") || canvas.getContext("experimental-webgl") || canvas.getContext("webkit-3d") || canvas.getContext("moz-webgl");
```

Comment:
```suggestion
  var gl = canvas.getContext("webgl2");
```

I would check directly against webgl2 without fallbacks, to ensure maximum compatibility

Reply:
Done in https://github.com/decentraland/explorer/pull/2472/commits/4be1aac39c3c677cbb336542a8ccf422829d64ec. Thanks!!